client/src/components/Donation/donation-modal.tsx

```diff
@@ -101,13 +102,29 @@ function DonateModal({
   location,
   recentlyClaimedBlock
 }: DonateModalProps): JSX.Element {
-  const [closeLabel, setCloseLabel] = React.useState(false);
-  const [ctaNumber, setCtaNumber] = React.useState(0);
+  const [closeLabel, setCloseLabel] = useState(false);
+  const [ctaNumber, setCtaNumber] = useState(0);
+  const [isDisabled, setIsDisabled] = useState(true);
+  const [showSkipButton, setShowSkipButton] = useState(false);
+  const loadElementsIdividually = useFeature('load_elements_individually').on;
   const { t } = useTranslation();
   const handleProcessing = () => {
     setCloseLabel(true);
   };
 
+  useEffect(() => {
+    if (!loadElementsIdividually) {
+      setIsDisabled(false);
+      setShowSkipButton(true);
+      return;
+    }
+    const timer = setTimeout(() => {
+      setIsDisabled(false);
+      setShowSkipButton(true);
+    }, 4000);
+    return () => clearTimeout(timer);
+  }, [loadElementsIdividually]);
```

Comment:
```suggestion
    if (loadElementsIndividually) {
      const timer = setTimeout(() => {
        setIsDisabled(false);
        setShowSkipButton(true);
      }, 4000);
      return () => clearTimeout(timer);
    } else {
      setIsDisabled(false);
      setShowSkipButton(true);
    }
  }, [loadElementsIndividually]);
```
Generally I think it's nicer to avoid negation if possible (less thinking required). Also, I like early returns, but I find they get less and less clear the further the `return` gets from the condition.